testing/MLDB-1428-text-sparse-output.py

```diff
@@ -84,4 +84,39 @@ def test_sparse_unknown_cols(self):
                     ["471242",1]]
         self.assertEqual(res, expected)
 
+    # MLDB-1513
+    def test_missing_rows(self):
```

Comment:
could those 2 test cases maybe be combined into one with a: for datasetType in ["tabular", "sparse.mutable"]: ?

then it would be clear that we're trying to control for identical behavior for two dataset types